config/azure/build.yaml

```diff
@@ -23,13 +23,24 @@ jobs:
 
    steps:
 
+   - task: UsePythonVersion@0
+     inputs:
+       versionSpec: '2.7'
+       addToPath: true
+       architecture: 'x64'
+     condition: eq( variables['Agent.OS'], 'Windows_NT' )
+
+   - script: python -m pip install scons
+     displayName: Install Scons
+     condition: eq( variables['Agent.OS'], 'Windows_NT' )
```

Comment:
Could we make this `Install Toolchain (Windows)` and also include the `doxygen` install here, instead of in `installDependencies.py`. We want to keep that just being the compile dependencies. 

We've been using that pattern on other platforms and in Gaffer too so it'll fit in nicely with those too then.

Reply:
Taken care of in fe1733e